flow/connectors/postgres/postgres.go

```diff
@@ -514,8 +517,8 @@ func (c *PostgresConnector) NormalizeRecords(req *model.NormalizeRecordsRequest)
 
 	return &model.NormalizeResponse{
 		Done:         true,
-		StartBatchID: batchIDs.NormalizeBatchID + 1,
-		EndBatchID:   batchIDs.SyncBatchID,
+		StartBatchID: normBatchID + 1,
```

Comment:
could we have a comment above briefly explaining why start and end batch IDs are set to these

Reply:
Or having it in the PR description would help too

Reply:
tbh it doesn't really matter, these responses aren't fed back into the system, only used for display

Reply:
thanks